app/src/main/java/com/example/voyagerx/LaunchDetailsFragment.kt

```diff
@@ -116,4 +117,17 @@ class LaunchDetailsFragment : Fragment() {
     private fun hideView(view: View) {
         view.visibility = View.GONE
     }
+
+    private fun shareLaunch() {
+        val share = Intent.createChooser(Intent().apply {
```

Comment:
I was thinking the chooser would be more complicated to do, this is pretty neat

Reply:
I know right! Super easy